_includes/boxes.html

```diff
@@ -13,18 +13,18 @@
 {% endif %}
 
 <div>
-    <div class="uk-container{{container}}">
+    <div class="uk-container{{container}} uk-container-expand">
 
         {% if include.title %}
-            <h2 class="uk-h1 uk-text-center">{{ include.title }}</h2>
+            <h2 class="uk-h1">{{ include.title }}</h2>
         {% endif %}
 
-        {% if include.subtitle %}<p class="uk-text-lead uk-text-center">{{ include.subtitle | escape }}</p>{% endif %}
+        {% if include.subtitle %}<p class="uk-text-lead">{{ include.subtitle | escape }}</p>{% endif %}
 
         <div class="uk-child-width-1-{{ columns }}@m uk-grid-match uk-text-center uk-margin-medium-top" data-uk-grid>
         {% for item in site.data.navigation_boxes %}
             {% if item.title %}
-            {% assign doc_url = item.doc | prepend:"/docs/" | append:"/" %}
```

Comment:
I don't quite understand why this is prepending `/docs/` (not all content is in there?), but assuming that is for a good reason!

Reply:
This is not yet finished. Was just for show and tell. I'll do a homepage PR that will fix this.